eg/Classes/MainFrame/__init__.py

```diff
@@ -521,12 +521,47 @@ def GetPanelDirection(self, panel):
         pos = info.find(";dir=") + 5
         return info[pos]
 
+    def SetWindowStyleFlag(self, *args, **kwargs):
+        """
+        Changes the main frame style depending on the display of the tray icon.
+
+        Sets the stlye flags fo the minimize button will be grayed out if the
```

Comment:
well hell I guess there was more then one typo LMAO

it's been one of those days..